san/batch.py

```diff
@@ -15,8 +16,14 @@ def execute(self):
         result = []
         batched_queries = []
 
-        for idx, q in enumerate(self.queries):
-            batched_queries.append(get_gql_query(idx, q[0], **q[1]))
+        for idx, query in enumerate(self.queries):
```

Comment:
This function is getting very complex. We can split it into smaller named functions.
lets have extract: 
```python
        batched_queries = []

        for idx, query in enumerate(self.queries):
            [metric, slug] = query[0].split('/')
            if metric in V2_METRIC_QUERIES:
                batched_queries.append(
                    san.sanbase_graphql.get_metric(
                        idx, metric, slug, **query[1]))
            else:
                batched_queries.append(get_gql_query(idx, query[0], **query[1]))

        gql_string = self.__batch_gql_queries(batched_queries)
```
into `__create_batched_query_string`.

So we end up with:
```
def execute(self):
    graphql_string = self.__create_batched_query_string()
    result = execute_gql(gql_string)
    return self.__transform_batch_result(result)
```

Reply:
It will be truly more readable, I will try and make it so if you wish. @IvanIvanoff  @kalkov What do you think?